src/App/PropertyLinks.h

```diff
@@ -183,6 +183,19 @@ class AppExport PropertyLinkBase : public Property, public ScopedLink
     virtual void getLinks(std::vector<App::DocumentObject *> &objs,
             bool all=false, std::vector<std::string> *subs=nullptr, bool newStyle=true) const = 0;
 
+    /** Obtain identifiers from this link property that link to a give object
+     * @param identifiers: holds the returned identifier to reference the given object
+     * @param obj: the referenced object
+     * @param subname: optional subname reference
+     * @param all: if true, then return all the references regardless of
+     *             this LinkScope. If false, then return only if the LinkScope
+     *             is not hidden.
+     */
+    virtual void getLinksTo(std::vector<App::ObjectIdentifier> &identifiers,
+                            App::DocumentObject *obj,
+                            const char *subname=nullptr,
+                            bool all=false) const = 0;

```

Comment:
As usual - bool arguments should be avoided.

Reply:
Agreed, but I'm not going to change the signatures of methods written by somebody else that are building blocks of the larger problem trying to be solved on a deadline.  This is 5 of 9-10 steps required, and fixing every one these uglies will take months, not days or weeks.  The cognitive overload of remembering and adjusting everything we've already changed in flight is already exceeding what a person can contain.